### ISSUE TITLE
Comments on the line with conditional compilation are not formatted correctly

### ISSUE THREAD

**Issue by bclothier** [WaynePhillipsEA/twinbasic#638]
**Describe the bug**
It is legal to add comment following a conditional compilation statement. However, they don't get formatted correctly.
![image](https://user-images.githubusercontent.com/2367644/148157624-d1440299-b101-471a-bd22-c061390432c8.png)

**To Reproduce**
1. Add code:
```
Public Module Example
    #If VBA7 Then 'I am a comment!
    #Else 'But why am I not a comment?
    #End If 'Nor I? 
End Module
```
2. Wait for formatting
3. Despair

**Expected behavior**
Comments should be comments.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - twinBASIC compiler version 0.13.62

**Comment by WaynePhillipsEA** [comment]
This is now fixed in v0.13.64 so that comments at the end of the conditional compilation statements will appear the same as if they were defined inside the block on the next line (either greyed out for dead blocks, or as green comments).

**Comment by bclothier** [comment]
![image](https://user-images.githubusercontent.com/2367644/148717473-0a677264-5462-499f-ac4c-5ee2e25e5411.png)

I just want to check --- is it reasonable to treat the comment after the `#Else` as dead? It makes sense for the contents inside the block but I'm not sure about the after the `#Else`.

Note that in VBA, it doesn't seem to be legal to have a colon to put more statements following the conditional compilation keywords. It seems to be legal in tB.
